package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Machine-to-machine interface for downloading form submissions",
   "repository": "git@github.com:cds-snc/forms-api.git",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc -p tsconfig.build.json && tsc-alias -p tsconfig.build.json",
```

Comment:
More information about why we have a `tsconfig.build.json` file in the comments attached to both `tsconfig.json` and `tsconfig.build.json` (below).